src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -224,7 +224,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
-    handleAnnotationStatusChange = (id: string, status: FeedItemStatus, permissions: AnnotationPermission) => {
+    handleAnnotationStatusChange: OnAnnotationStatusChange = ({ id, permissions, status }) => {
         const { api, emitAnnotationUpdateEvent, file } = this.props;
```

Comment:
I'd recommend not using this and doing the same way you did for onCommentStatusChangeHandler in ActiveState. The folks from Poland put this here before for their ActivityThread implementation but that won't be used anymore once we push out our code. It's best stay as consistent as possible between annotations and comments functionality. What are your thoughts on this?

Reply:
Im open to going that route. If I do that, I think the `handleAnnotationStatusChange` and `handleAnnotationEdit` should be collapsed into one function. I can conditionally build that `description` object when `text` is passed in. If they are not combined back together, I think it makes more sense to use the function that specifically does this.

Reply:
After discussion, we will proceed with the current code structure in the scope of this PR. A new ticket will be opened to handle standardising comments and annotations across all component.